### PR TITLE
fix(holocron module): moduleLoadStatus 'loading' even with no load or loadModuleData functions passed #63

### LINKED ISSUE #63: If loadModuleData is not passed to Module.holocron, then moduleLoadStatus is always 'loading'
# 🐞 Bug Report

## Describe the bug

If neither 'load' nor 'loadModuleData' are passed to [Module].holocron, the module's 'moduleLoadStatus' is 'loading'.

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Run the one-app generator command: npx -p yo -p @americanexpress/generator-one-app-module -- yo @americanexpress/one-app-module
2. Edit the generated function in the src/components/[name].jsx file to deconstruct 'moduleLoadStatus' from the arguments.
3. Display/console.log the moduleLoadStatus (it will be loading)

## Expected behavior

If there is no asynchronous data to load, 'moduleLoadStatus' should be 'loaded'.

## System information

- OS: Windows (64-bit)
- Browser: N/A
- Version of holocron: ^1.1.0
- Node version: 12.18.1

## Additional context

You can copy/paste this code into the .jsx to easily reproduce:

```javascript
import React from 'react';
import PropTypes from 'prop-types';
import childRoutes from '../childRoutes';

const HelloWorld = ({ moduleLoadStatus }) => {
  if (moduleLoadStatus === 'loading') return <h1>Loading...</h1>; // <-- This will always be returned!
  if (moduleLoadStatus === 'error') return <h1>Error!</h1>;
  return <h1>Hello World!</h1>;
};

HelloWorld.childRoutes = childRoutes;

HelloWorld.propTypes = {
  moduleLoadStatus: PropTypes.oneOf(['loading', 'loaded', 'error']),
};

HelloWorld.holocron = {
  name: 'hello-world',
};

if (!global.BROWSER) {
  // eslint-disable-next-line global-require
  HelloWorld.appConfig = require('../appConfig').default;
}

export default HelloWorld;
```

### PR DESCRIPTION
**Pull request by AdamFoskit**
Fix the problem where `moduleLoadStatus` was loading, even if neither `load` nor `loadModuleData` were passed to the holocron module.

## Description
Removed the if statement, as suggested [here](https://github.com/americanexpress/holocron/issues/63#issuecomment-747638073).

## Motivation and Context
Fixes #63 

## How Has This Been Tested?
Tested with the one-app module serving a holocron package locally.

## Types of Changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation (adding or updating documentation)
- [ ] Dependency update

## Checklist:
- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [x] My changes are in sync with the code style of this project.
- [x] There aren't any other open Pull Requests for the same issue/update.
- [ ] These changes should be applied to a maintenance branch.
- [ ] This change requires cross browser checks.
- [ ] This change impacts caching for client browsers.
- [ ] This change adds additional environment variable requirements for Holocron users.
- [ ] I have added the Apache 2.0 license header to any new files created.

## What is the Impact to Developers Using Holocron?
This should not have any negative impact on developers currently using holocron.

### CODE REVIEW

**Comment by AdamFoskit** [comment]
Also, when commiting this change from a Windows machine I needed to change the "test:unit" script [here.](https://github.com/americanexpress/holocron/blob/86910f2167884736e108078bb4f01bf6abb883c8/package.json#L22)
See [this](https://github.com/americanexpress/holocron/issues/67).

**Comment by JAdshead** [comment]
@AdamFoskit you can add cross-env to make it work on both windows and mac - https://www.npmjs.com/package/cross-env

**Review by JAdshead — APPROVED** [review]

**Comment by AdamFoskit** [comment]
@JAdshead Have you used cross-env on a Mac? About to test it locally on my Windows machine.

**Comment by JAdshead** [comment]
> @JAdshead Have you used cross-env on a Mac? About to test it locally on my Windows machine.

yep we use it on a few of our other repos

File: packages/holocron/docs/api/README.md [hunk 1]
```diff
@@ -140,7 +140,18 @@ HelloWorld.holocron = {
   shouldModuleReload,
   // options,
 };
+<<<<<<< refs/remotes/upstream/main
```
Comment (infoxicator) [inline packages/holocron/docs/api/README.md:143]: please fix conflicts 👍

**Review by JAdshead — APPROVED** [review]

**Review by jgolden17 — APPROVED** [review]

**Review by infoxicator — APPROVED** [review]

**Review by Francois-Esquire — APPROVED** [review]

**Pull request merged**